tests/testthat/test-is-study-name-valid.R

```diff
@@ -0,0 +1,41 @@
+context("test-is-study-name-valid.R")
+
+test_that("is_study_name_valid returns TRUE when name is valid", {
+  name1 <- "study"
+  name2 <- "study name"
+  name3 <- "study_name"
+  name4 <- "study (name)"
+  name5 <- "study.name"
+  name6 <- "study-name"
+  name7 <- "study1"
+  name8 <- "study+"
+  expect_true(is_study_name_valid(name1))
+  expect_true(is_study_name_valid(name2))
+  expect_true(is_study_name_valid(name3))
+  expect_true(is_study_name_valid(name4))
+  expect_true(is_study_name_valid(name5))
+  expect_true(is_study_name_valid(name6))
+  expect_true(is_study_name_valid(name7))
+  expect_true(is_study_name_valid(name8))
+})
+
+test_that("is_study_name_valid returns FALSE when name is invalid", {
```

Comment:
Can we add one test for the study name `""`?

Reply:
Haha, whoops. Thank you for catching that! Added.